tests/Functional/Schema/MySQLSchemaManagerTest.php

```diff
@@ -394,6 +394,11 @@ public function testJsonColumnType(): void
         $table->addColumn('col_json', Types::JSON);
         $this->dropAndCreateTable($table);
 
+        // Remove the comment from the column to ensure the type is detected correctly from the check constraints.
+        if ($this->connection->getDatabasePlatform() instanceof MariaDBPlatform) {
+            $this->connection->executeStatement('ALTER TABLE test_mysql_json CHANGE COLUMN col_json col_json JSON');
+        }
```

Comment:
Do we still need this? 4.0 should ignore those column comments.

Reply:
Done dfe3145abe5ed004879a9f93dedf84180f660b8e